src/client.tsx

```diff
@@ -11,6 +12,12 @@ const render = (Component: React.FC) => {
 const initializeApp = () => {
     const App = require('./components/App/App').App;
 
+    const { ENABLE_GA, GA_TRACKING_ID } = env;
+
+    if (ENABLE_GA) {
```

Comment:
All env vars come back as strings so this will always be true if the var exists.  That means you'll need to do an explicit check here for either `!="false" || !="0"`.